ci/pipeline.yml

```diff
@@ -240,4 +378,36 @@ resource_types:
   - name: slack-notification
     type: docker-image
     source:
-      repository: cfcommunity/slack-notification-resource
\ No newline at end of file
+      repository: cfcommunity/slack-notification-resource
+
+  - name: pull-request
+    type: docker-image
+    source:
+      repository: teliaoss/github-pr-resource
+
+############################
+# Groups - Use to better visualize and track jobs in CI
```

Comment:
Haven't seen `groups` before, do we use these elsewhere?

Reply:
Not in Pages but we do in cloud-gov. We don't have too complex of pipelines but it maybe nice to visualize the deployments while we straddle Pages and Federalist.